pkg/registry_checker/checker.go

```diff
@@ -325,7 +326,7 @@ func check(ref name.Reference, kc *keychain, registryTransport *http.Transport)
 
 		kc.index = 0
 	} else {
-		_, imgErr = remote.Head(ref, remote.WithTransport(registryTransport), remote.WithContext(ctx))
```

Comment:
this is now changed upstream, move the `authn.DefaultKeychain` to https://github.com/deckhouse/k8s-image-availability-exporter/blob/master/pkg/registry_checker/indexers.go#L282 and replace the nil return.